src/Services/FileBlogService.cs

```diff
@@ -109,7 +125,7 @@ public virtual Task<IEnumerable<string>> GetCategories()
                             ));
 
             XElement categories = doc.XPathSelectElement("post/categories");
-            foreach (string category in post.Categories)
+            foreach (string category in post.PostCategories.Select(p => p.CategoryID))
```

Comment:
Perhaps the "proper" thing to do here would be to use `post.PostCategories.Select(p => p.Category.Id)`; however, it seems really unnecessary (to the extent that I'm pretty sure I left `p.Category` null). If Category ever expanded than, of course this would need to change.